app/controllers/concerns/my_facilities_filtering.rb

```diff
@@ -38,11 +38,7 @@ def set_only_new_facilities
     end
 
     def facilities_by_size(facilities)
-      if (@facility_sizes - @selected_sizes).empty?
-        facilities
-      else
-        facilities.where(facility_size: @selected_sizes)
-      end
+      facilities.where(facility_size: @selected_sizes)
```

Comment:
@claudiovallejo Facility size was recently made a mandatory field, so there's still old data without any `size` set. I think we'll have to keep the `if (@facility_sizes - @selected_sizes).empty?` clause for when all sizes are selected.